SIA.tex

```diff
@@ -104,26 +74,27 @@ \subsubsection{Simple Data Discovery}
+The data discovery phase returns a subset of the available metadata.  Clients may need additional detailed metadata (as defined by any IVOA Data model specification) in order to make decisions or perform computations required to access the data (e.g. using a separate low-level data access service as described in the SODA specification). The client must be able to easily figure out if detailed metadata is available and, using an identifier from the discovery response, make a call to a web service to retrieve the detailed metadata.
 
 \subsubsection{Download Complete Datasets}
 \label{sec:sync}
 
 The client should be able to download complete datasets with information available in the discovery response. If the dataset is a single file, the service should provide an access URL to the file; if it is multiple files, then an access URL to a DataLink service [9] can be provided, but the client must be able to easily distinguish these two scenarios.
 
-\subsubsection{Access a Datacube with Operations: Too Big to Download}
+\subsubsection{Access a Dataset with Operations: Too Big to Download}
 \label{sec:async}
-In many cases, datacubes are too large to download and process locally, so the client must be able to perform remote operations. Data discovery could be performed using any discovery protocol (SIA, TAP with ObsCore, etc.).  The client must be able to easily figure out if a low level access service is available for a discovered dataset. This could be using a URL provided in the response or by calling an associated DataLink service. Access operations include basic filtering (cut out a subsection of the data), transformations, or other pixel-level operations or even analysis. With current  version of the AccessData specification, we will only cover extracting a simple subset of an image or datacube. 
+In many cases, datasets are too large to download and process locally, so the client must be able to perform remote operations. Data discovery could be performed using any discovery protocol (DsSAP, SIA, TAP with ObsCore, etc.).  The client must be able to easily figure out if a low level access service is available for a discovered dataset. This could be using a URL provided in the response or by calling an associated DataLink service. Access operations include basic filtering (cut out a subsection of the data), transformations, or other pixel-level operations or even analysis. With version 1.0 of the SODA 1.0  specification, we will only cover extracting a simple subset of an image or datacube. 
```

Comment:
The 1.0 in SODA 1.0 is repeated and should this be replaced with a citation of SODA 1.0?

Reply:
OK. Remove second 1.0